agent/config/config.go

```diff
@@ -434,6 +445,25 @@ func getTaskCPUMemLimitEnabled() Conditional {
 	return taskCPUMemLimitEnabled
 }
 
+func getTaskMetatadataRpsLimits() (int, int) {
+	var rpsLimit, rpsBurstLimit int
+	rpsLimitEnvVal := os.Getenv("ECS_TASK_METADATA_RPS_LIMIT")
+	rpsLimitSplits := strings.Split(rpsLimitEnvVal, ",")
+	if len(rpsLimitSplits) != 2 {
+		seelog.Warnf("Invalid format for \"ECS_TASK_METADATA_RPS_LIMIT\", expected: \"rateLimit,burst\"")
+	} else {
+		rpsLimitVal, err := strconv.Atoi(strings.TrimSpace(rpsLimitSplits[0]))
+		rpsBurstLimitVal, err1 := strconv.Atoi(strings.TrimSpace(rpsLimitSplits[1]))
+		if err != nil || err1 != nil {
```

Comment:
This kind of error handling makes it harder to know which field exactly caused the error.  Doing something like this is much simpler imo:

```
rpsLimit, err := strconv.Atoi(strings.TrimSpace(rpsLimitSplits[0]))
if err != nil {
 ...
 return 0, 0
}

rpsBurstLimit, err := strconv.Atoi(strings.TrimSpace(rpsLimitSplits[1]))
if err != nil {
 ...
 return 0, 0
}

return rpsLimit, rpsBurstLimit
```